### DISCUSSION TITLE
Add option to open a task as a drawing for pen use.

### DISCUSSION THREAD

**Discussion by tiptruck** [TechbeeAT/jtxBoard#1227] [Ideas]
Hi

I have put this in Discussions because it may go outside the goals of JTX.

Many tablets now use a pen to input data.  Similar to the recording option when opening a quick task, is it possible to add an option to the quick task screen to open a blank drawing as a task? I can then write on it like I would in other handwriting apps.  This really comes into its own for writing maths in the notes, at the moment I use another note taking app for these occassions.

This could also turn into a real can of worms, with users requesting more and more professional drawing tools.

Regards Craig

**Comment by patrickunterwegs** [comment]
Hi @tiptruck , 
thanks, indeed I've also been thinking about it. As you already say, users could start requesting more and more regarding this feature, so I'm a bit hesitating, but still, if I find a good library that does the job well enough, I'd consider it. But for now I still have other priorities.
